Library/Config.ts

```diff
@@ -8,37 +8,64 @@ class Config {
     // This key is provided in the readme
     public static ENV_iKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
     public static legacy_ENV_iKey = "APPINSIGHTS_INSTRUMENTATION_KEY";
+    public static ENV_profileQueryEndpoint = "APPINSIGHTS_PROFILE_QUERY_ENDPOINT";
 
     public instrumentationKey: string;
-    public instrumentationKeyHash: string;
+    public correlationId: string;
     public sessionRenewalMs: number;
     public sessionExpirationMs: number;
+    public endpointBase: string;
```

Comment:
Internal users can easily set both endpointUrl and profileQueryEndpoint. IMO endpointBase can just be a private const used to avoid duplicating the default string